src/classes/UTIL_Query.cls

```diff
@@ -44,20 +44,23 @@ public with sharing class UTIL_Query {
     /** @description Error message when FROM value is not provided */
     private static final String FROM_ENTITY_REQUIRED = 'The FROM value is required.';
 
+    /** @description SELECT field to be returned if requested and the Org is a Multi Currency Enabled Org */
+    public static final String CURRENCYISOCODE_FIELD = 'CurrencyIsoCode';
```

Comment:
You can use `UTIL_Currency.CURRENCY_ISO_CODE_FIELD` for the method below rather than defining a new constant